COMPILATION.md

```diff
@@ -15,7 +15,9 @@ Keep in mind using the pre-built packages when available.
 * libcurl
 * libxml2
 * openssl
-* /etc/mime.types (the package providing depends on the OS)
+* mime.types (the package providing depends on the OS)
+	* As default, the file is searched in the order /etc/mime.types to /etc/apache2/mime.types, and first detected file is loaded as mime information.
```

Comment:
It seems this needs an update:
> A `mime.types` file (the package providing it depends on the OS)
> * As default, `/etc/mime.types` is checked (ff this file does not exist on macOS, then `/etc/apache2/mime.types` is checked as well)
> * Alternatively, Set `mime.types` file path with `mime` option, it is loaded instead of default search files.

Reply:
I'm sorry that I forgot to modify this file.
I updated this file.

Reply:
I'd recommend highlighting filenames and paths as I did at the quote above.

Reply:
Thanks for review.
I set highlighting paths and updated file(with squash)